.pipelines/mirror-aro-to-int.yml

```diff
@@ -4,11 +4,11 @@ trigger: none
 resources:
   pipelines:
   - pipeline: mirror
-    source: Build and Push RP image tagged
+    source: "Build and Push RP image tagged"
```

Comment:
I think the root issue is you can only login with one service principal at a time. Therefore, you cannot access both an INT and PROD ACR while logged in as a single service principal. I'm not confident this change would fix the error.

I think it would be the same error, but shifted from failing to authenticate to PROD to failing to authenticate to INT ACR.